content/rs/references/rest-api/requests/actions.md

```diff
@@ -0,0 +1,142 @@
+---
+Title: Actions requests
+linkTitle: actions
+description: Actions requests
+weight: $weight
+alwaysopen: false
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases: /rs/references/rest-api/actions
+         /rs/references/rest-api/actions.md
+         /rs/references/restapi/actions
+         /rs/references/restapi/actions.md
+         /rs/references/rest_api/actions
+         /rs/references/rest_api/actions.md
+---
+
+| Method | Path | Description |
+|--------|------|-------------|
+| [GET](#get-all-actions) | `/v1/actions` | Get all actions |
+| [GET](#get-action) | `/v1/actions/{uid}` | Get a single action |
+
+## Get all actions {#get-all-actions}
+
+```
+GET /v1/actions
+```
+
+Get the status of all currently executing, queued, or completed actions on all entities (clusters, nodes, and databases). This API is used to track every long-lived API request which returns either a `task_id` or an `action_uid`.
```

Comment:
```suggestion
Get the status of all actions (executing, queued, or completed) on all entities (clusters, nodes, and databases). This API tracks long-lived API requests return either a `task_id` or an `action_uid`.
```

Reply:
```suggestion
Get the status of all actions (executing, queued, or completed) on all entities (clusters, nodes, and databases). This API tracks long-lived API requests that return either a `task_id` or an `action_uid`.
```
Made a minor change to the second sentence here - let me know what you think.